src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexNode.cs

```diff
@@ -1210,7 +1210,6 @@ static RegexNode ExtractCommonPrefixText(RegexNode alternation)
                 }
 
                 Span<char> scratchChar = stackalloc char[1];
```

Comment:
```suggestion
                char c = default;
                Span<char> scratchChar = new Span<char>(ref c);
```

Are we trying to avoid the initialization cost?

Reply:
This file is shared by a netstandard2.0 project; it can't use this Span ctor.